server/schemas/resolvers.js

```diff
@@ -38,12 +36,17 @@ const resolvers = {
     },
 
     addParkingPlace: async (parent, args, context) => {
-      console.log(context);
       if (context.user) {
-        // const parkingLoation = await ParkingPlace.create({provider:context.user._id} , args);
-        // if(parkingLoation.length>0){
-        //   return await Inventory.create(parkingLoation._id,args.startDate ,args.endDate );
-        // }
+        const parkingLot = await ParkingPlace.create({
+          ...args,
+          provider: context.user._id,
+        });
+        await User.findByIdAndUpdate(
+          { _id: context.user._id },
+          { $push: { parkingPlace: parkingLot._id } }
+        );
```

Comment:
Here ParkingPlace  is getting used to User table than we don't need providerID in ParkingPlace table. We might need to look model again.